src/trafficgen/read_files.py

```diff
@@ -1,251 +1,249 @@
-
-    Params:
-        own_ship_file: Path to the own_ship_file file
-
-    Returns
-    -------
-        own_ship information
-    """
-    assert settings.classification is not None
-
-    settings.classification.theta13_criteria = deg_2_rad(settings.classification.theta13_criteria)
-    settings.classification.theta14_criteria = deg_2_rad(settings.classification.theta14_criteria)
-    settings.classification.theta15_criteria = deg_2_rad(settings.classification.theta15_criteria)
-    settings.classification.theta15[0] = deg_2_rad(settings.classification.theta15[0])
-    settings.classification.theta15[1] = deg_2_rad(settings.classification.theta15[1])
-
-    settings.vector_range[0] = min_2_s(settings.vector_range[0])
-    settings.vector_range[1] = min_2_s(settings.vector_range[1])
-
-    settings.situation_length = min_2_s(settings.situation_length)
-    settings.max_meeting_distance = nm_2_m(settings.max_meeting_distance)
-    settings.evolve_time = min_2_s(settings.evolve_time)
-
-    return settings
-
-
-def check_input_units(data: Dict[str, Any]) -> Dict[str, Any]:
-    """Check if input unit is specified, if not specified it is set to SI."""
-
-    if "input_units" not in data:
-        data["input_units"] = "si"
-
-    return data
-
-
-def camel_to_snake(string: str) -> str:
-    """Convert a camel case string to snake case."""
-    return "".join([f"_{c.lower()}" if c.isupper() else c for c in string]).lstrip("_")
-
-
-def convert_keys_to_snake_case(data: Dict[str, Any]) -> Dict[str, Any]:
-    """Convert keys in a nested dictionary from camel case to snake case."""
-    converted_dict = {}
-    for key, value in data.items():
-        converted_key = camel_to_snake(key)
-        if isinstance(value, dict):
-            converted_value = convert_keys_to_snake_case(value)
-        elif isinstance(value, list):
-            converted_value = convert_list_of_dict_to_snake_case(value)
-        else:
-            converted_value = value
-        converted_dict[converted_key] = converted_value
-    return converted_dict
-
-
-def convert_list_of_dict_to_snake_case(data: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
-    """Loops through a list of dics to convert keys from camel case to snake case."""
-
-    converted_list: List[Dict[str, Any]] = []
-    for item in data:
-        if not isinstance(item, dict):
-            return data
-        converted_item = convert_keys_to_snake_case(item)
-        converted_list.append(converted_item)
-    return converted_list
+"""Functions to read the files needed to build one or more traffic situations."""
+
+import json
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Union
+from uuid import UUID, uuid4
+
+from trafficgen.types import (
+    EncounterSettings,
+    UnitType,
+)
+from trafficgen.utils import deg_2_rad, knot_2_m_pr_s, min_2_s, nm_2_m
+
+from maritime_schema.types.caga import AISNavStatus, OwnShip, TargetShip, TrafficSituation

```

Comment:
(same) -> brackets, trailing comma -> auto-format :-)

(one list item per line makes sense for longer lists, because then changes in the list translate to added / deleted lines in version control, which is easy to visually identify and "monitor" in a diff view between commits.